packages/design-system/src/components/web-components/ds-alert.test.tsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import Alert, { AlertProps } from '../Alert/Alert';
+import { UtagContainer } from '../analytics';
+import { setAlertSendsAnalytics } from '../flags';
+import { render, screen } from '@testing-library/react';
+import register from 'preact-custom-element';
+
+register(Alert, 'ds-alert');
```

Comment:
i tried using the `define()` from preactement and it resulted in no passing tests compared to using `register()` from preact-custom-element. not sure what's going on here, but perhaps this is something to look into with the next phase?